docs/operations_table.py

```diff
@@ -0,0 +1,119 @@
+from nvidia.dali import backend as b
+import nvidia.dali.ops as ops
+import nvidia.dali.fn as fn
+import nvidia.dali.plugin.pytorch
+import sys
+
+# Dictionary with modules that can have registered Ops
+ops_modules = {
+    'nvidia.dali.ops': nvidia.dali.ops,
+    'nvidia.dali.plugin.pytorch': nvidia.dali.plugin.pytorch
+}
+
+cpu_ops = ops.cpu_ops()
+gpu_ops = ops.gpu_ops()
+mix_ops = ops.mixed_ops()
+all_ops = cpu_ops.union(gpu_ops).union(mix_ops)
+link_formatter = ':meth:`{op} <{module}.{op}>`'
+
+def to_fn_name(full_op_name):
+    tokens = full_op_name.split('.')
+    tokens[-1] = fn._to_snake_case(tokens[-1])
+    return '.'.join(tokens)
+
+def name_sort(op_name):
+    _, module, name = ops._process_op_name(op_name)
+    return '.'.join(module + [name.upper()])
+
+def longest_fn_string():
+    longest_str = ""
+    for op in sorted(all_ops, key=name_sort):
+        fn_string = ""
+        op_full_name, submodule, op_name = ops._process_op_name(op)
+        for (module_name, module) in ops_modules.items():
+            m = module
+            for part in submodule:
+                m = getattr(m, part, None)
+                if m is None:
+                    break
+            if m is not None and hasattr(m, op_name):
+                fn_string = link_formatter.format(op = to_fn_name(op_full_name), module = module_name.replace('.ops', '.fn'))
+                if len(fn_string) > len(longest_str):
+                    longest_str = fn_string
+    return longest_str
+
+op_name_max_len = len(longest_fn_string())
+name_bar = op_name_max_len * '='
+
+def fn_to_op_table(out_filename):
+    formater = '{:{c}<{op_name_max_len}} {:{c}<{op_name_max_len}}\n'
+    doc_table = ''
+    doc_table += formater.format('', '', op_name_max_len = op_name_max_len, c='=')
+    doc_table += formater.format('Function (fn.*)', 'Operator Object (ops.*)', op_name_max_len = op_name_max_len, c=' ')
+    doc_table += formater.format('', '', op_name_max_len = op_name_max_len, c='=')
+    for op in sorted(all_ops, key=name_sort):
+        op_full_name, submodule, op_name = ops._process_op_name(op)
+        schema = b.TryGetSchema(op)
+        if schema:
+            if schema.IsDocHidden():
+                continue
+        for (module_name, module) in ops_modules.items():
+            m = module
+            for part in submodule:
+                m = getattr(m, part, None)
+                if m is None:
+                    break
+            if m is not None and hasattr(m, op_name):
+                op_string = link_formatter.format(op = op_full_name, module = module_name)
+                fn_string = link_formatter.format(op = to_fn_name(op_full_name), module = module_name.replace('.ops', '.fn'))
```

Comment:
Maybe add `to_fn_module`, I see that `module_name.replace('.ops', '.fn')` appears several times in this file.